powerauth-webflow-authentication/src/main/java/io/getlime/security/powerauth/lib/webflow/authentication/configuration/WebFlowServicesConfiguration.java

```diff
@@ -223,6 +229,14 @@ public AfsType getAfsType() {
         return afsType;
     }
 
+    /**
+     * Get whether client IP address is detected in anti-fraud system integration.
+     * @return Whether client IP address is detected in anti-fraud system integration.
+     */
+    public boolean getAfsDetectIpAddress() {
```

Comment:
I would recommend to use `isXXX` or `hasXXX` for a boolean getter. Maybe change the field name a little bit?

`afsIpAddressDetectionEnabled` (too long)

Reply:
This variable name is too long.